examples/tutorials/advanced/contour_map.py

```diff
@@ -57,7 +53,7 @@
 fig = pygmt.Figure()
 fig.grdcontour(
     annotation=1000,
-    interval=250,
+    levels=250,
     grid=grid,
     limit=[-4000, -2000],
 )
```

Comment:
This code block and many others can be rewritten into a single line after we set the line length limit to 88 characters.

Reply:
Yes, I should find the time to make both updates. Should this be included in this PR, or better in separate PR and have this PAR for the deprecation?

Reply:
OK. Let's keep this PR small.

Reply:
Just submitted PR #3212 for working on this.